src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -49,6 +73,15 @@ public void stop() {
         io.setVoltage(0.0, 0.0);
     }
 
+    public void aim() {
+        this.forward = 0;
+        this.rotation = targetHeading.getAsDouble();
```

Comment:
The reason the robot is turning the same direction every time is because we are passing in the target heading into Arcade Drive directly.

It would be better to use a PID controller that changes this rotation variable based on the error between target heading and current heading. Make sure to enable continuous input in the PID